completion/available/aliases.completion.bash

```diff
@@ -50,7 +50,7 @@ function _bash-it-component-completion-callback-on-init-aliases() {
 		fi
 
 		# skip aliases to pipes, boolean control structures and other command lists
-		chars='\|\&\;\)\(\n\<\>'
+		chars=$'\|\&\;\)\(\n\<\>'
```

Comment:
Although this does fix the bug (good find!), it turns out the `\` aren't needed in this context at all, and this essentially just adds the `\` character to the match list (6 times!) ... Assuming we don't care to _actually_ match `\`, this is sufficient:

```
chars=$'|&;()<>\n'
```
And *much* easier to read.
I'm going to recommend we update this PR to the more simple string.